Docker/Dockerfile-build

```diff
@@ -0,0 +1,10 @@
+FROM debian
+
+RUN apt-get update && apt-get install -y\
+	gcc\
+	g++\
+	git\
```

Comment:
ez minek?

Reply:
Ezt sem igazán értem, mi minek? Egy teljesen jól működő Docker image készítéséhez amelyen build-elünk ezek a csomagok kellenek (ha fenn is vannak alapból, frissíteni nem ártott őket. Pl a gcc az alap Debian-okon is van mert pl a saját kernelmoduljait azzal fordítja a rendszer, de frissíteni sosem árt)

Reply:
a git sorara irtam, hogy minek :-)

Reply:
@adipapp  ezzel mi lesz? :-)

Reply:
Mivel történik egy checkout az adott job-ban (igazából mindegyikben) ezért kell neki git.

Reply:
oh, true, azt nem a workspace-bol rantjatok elo egy initial checkout utan. Akkor sajna muszaj bloatolni vele az image-et.